test/testinfra/MUXTestInfra/Common/InputHelper.cs

```diff
@@ -295,6 +309,74 @@ public static void TapAndHold(UIObject obj, uint durationMs)
             Wait.ForIdle();
         }
 
+        public static void DoubleTap(UIObject obj)
+        {
+            // Allowing three attempts to work around occasional failure on Phone builds.
```

Comment:
'on Phone builds'.  Maybe it's time to drop this part.

Reply:
Good point.